File: _Basic/windows-forms-introduction-with-visual-basic/1-windows-forms-introduction-with-visual-basic-m1-forms1-exercise-files/CS/Simple/Simple/MyForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Simple
{
    public class MyForm : Form
    {
        private TextBox MessageTextBox;
        private Button ShowMessageButton;
        private Label MessageLabel;

        public MyForm()
        {
            this.Text = "My Form";

            MessageTextBox = new TextBox();
            MessageTextBox.Left = 25;
            MessageTextBox.Top = 25;
            MessageTextBox.Width = 200;
            this.Controls.Add(MessageTextBox);

            ShowMessageButton = new Button();
            ShowMessageButton.Left = 25;
            ShowMessageButton.Top = 75;
            ShowMessageButton.Width = 200;
            ShowMessageButton.Text = "Show Message";
            this.Controls.Add(ShowMessageButton);
            ShowMessageButton.Click += ClickHandler;

            MessageLabel = new Label();
            MessageLabel.Left = 25;
            MessageLabel.Top = 125;
            MessageLabel.Width = 200;
            MessageLabel.Text = "[Label]";
            this.Controls.Add(MessageLabel);
        }

        private void ClickHandler(object sender, EventArgs e)
        {
            MessageLabel.Text = MessageTextBox.Text;
        }
    }
}

File: _Basic/windows-forms-introduction-with-visual-basic/1-windows-forms-introduction-with-visual-basic-m1-forms1-exercise-files/VB/Simple/Simple/MyForm.vb
Imports System.Windows.Forms

Public Class MyForm
    Inherits Form

    Private MessageTextBox As TextBox
    Private MessageLabel As Label
    Private WithEvents ShowMessageButton As Button

    Public Sub New()
        Me.Text = "My Form"

        MessageTextBox = New TextBox()
        MessageTextBox.Left = 25
        MessageTextBox.Top = 25
        MessageTextBox.Width = 200
        Me.Controls.Add(MessageTextBox)

        ShowMessageButton = New Button()
        ShowMessageButton.Left = 25
        ShowMessageButton.Top = 75
        ShowMessageButton.Width = 200
        ShowMessageButton.Text = "Show Message"
        Me.Controls.Add(ShowMessageButton)

        MessageLabel = New Label()
        MessageLabel.Left = 25
        MessageLabel.Top = 125
        MessageLabel.Width = 200
        MessageLabel.Text = "[Label]"
        Me.Controls.Add(MessageLabel)
    End Sub

    Private Sub ClickHandler(sender As Object, e As EventArgs) Handles ShowMessageButton.Click
        MessageLabel.Text = MessageTextBox.Text
    End Sub
End Class

